src/helpers.ts

```diff
@@ -90,12 +90,18 @@ export function createDefaultClient(
 }
 
 export async function createDefaultEthSignClientAsync(
-  dappchainKey: string,
   dappchainEndpoint: string,
   chainId: string,
   wallet: ethers.Signer
 ): Promise<{ client: Client; publicKey: Uint8Array; address: Address }> {
-  const defaultClientObj = createDefaultClient(dappchainKey, dappchainEndpoint, chainId)
+  const privKey = CryptoUtils.generatePrivateKey()
+
+  const defaultClientObj = createDefaultClient(
+    Uint8ArrayToB64(privKey),
```

Comment:
So we create a default client with a key we don't use, with middleware we don't use... might as well just copy the bits we do use from `createDefaultClient` into this function because clearly `createDefaultClient` really wasn't intended for this.